package.json

```diff
@@ -58,5 +58,20 @@
     "./**/*.{js,jsx,ts,tsx,scss}": [
       "npm run format"
     ]
-  }
+  },
+  "description": "This application enables transaction among members of RDS group. We have three different currencies : Bronze , Silver and Gold and these carry different values. You can shop items and acknowledge people for their work",
+  "main": ".babelrc.js",
+  "directories": {
+    "test": "test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/JoseLRiano/website-crypto.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/JoseLRiano/website-crypto/issues"
+  },
+  "homepage": "https://github.com/JoseLRiano/website-crypto#readme"
 }
```

Comment:
I don't think we need to update package.json for this change. Please remove these changes